valhalla/midgard/util_core.h

```diff
@@ -0,0 +1,32 @@
+#pragma once
```

Comment:
was this pulled out to aid in compilation times or something? i cant quite connect the dots on why its not in `util.h`